docs/styles/_contribute_list.scss

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Yugabyte, Inc.
+
+@import "./_colors.scss";
+
+#contribute-list{
```

Comment:
As we just talked about, I think the dropdown needs a little more contrast to help it get noticed. Nothing outrageous, but a bit more noticeable would be good.

Reply:
I have set the opacity to `0.2` for the dropdown, if you think its bright, i can change it to `0.1`

Reply:
Let's go with 0.2 for now. 0.1 feels too light, and 0.15 feels too nitpicky :)